src/base.cls.php

```diff
@@ -432,6 +433,7 @@ class Base extends Root
 		self::O_OPTM_JS_EXC => array(),
 		self::O_OPTM_HTML_MIN => false,
 		self::O_OPTM_HTML_LAZY => array(),
+		self::O_OPTM_HTML_SKIP_COMMENTS => '',
```

Comment:
L435 is an array(). Consider using that way.

Reply:
done